minimint/src/net/connect.rs

```diff
@@ -76,6 +79,11 @@ where
         Connections { connections: peers }
     }
 
+    pub fn drop_peer(&mut self, peer: &PeerId) {
+        self.connections.remove(peer);
```

Comment:
~I'd expect this to currently crash? If a peer disconnects earlier than others and the disconnected peer panics in response others who didn't disconnect yet will also panic. My upcoming network stack PR will fix this.~

I saw you fixed that

Reply:
Yeah, I think long-term we should persist all failed `send` messages to disk so if a peer reconnects we can replay them to catch the peer up.